SpatialGDK/Source/SpatialGDKEditor/Private/Utils/TransientUObjectEditor.cpp

```diff
@@ -0,0 +1,167 @@
+#include "Utils/TransientUObjectEditor.h"
+
+#include "PropertyEditor/Public/PropertyEditorModule.h"
+#include "MainFrame/Public/Interfaces/IMainFrameModule.h"
+
+#include "SlateApplication.h"
+#include "SBorder.h"
+#include "SButton.h"
+//#include "SDetailView.h"
+
+//DECLARE_LOG_CATEGORY_EXTERN(LogSpatialGDKTransientUObjectEditor, Log, All);
+
+static void OnTransientUObjectEditorWindowClosed(const TSharedRef<SWindow>& Window, UTransientUObjectEditor* Instance)
```

Comment:
Should probs have private declares for these in the header no?

Reply:
Not as long as these callbacks do not try to access the instance's internals.